### ISSUE TITLE
Add ability to use snackbar for Ng auth login error screen

### ISSUE THREAD

**Issue by emclaug2** [brightlayer-ui/angular-workflows#127]
#### Describe the desired behavior
Add snackbar as an option in addition to text field error, message box and dialog. 

#### Describe the current behavior
Snackbar is not supported as a error message variant.

#### Is this request related to a current issue?
No

#### Additional Context

No

**Comment by bkarambe** [comment]
@huayunh Do we have designs for how this will look like?

**Comment by huayunh** [comment]
No; although a snack bar is pretty standard. You can just throw one in anchored at the bottom center of the page
